hr_expense_receipt_require/models/hr_expense.py

```diff
@@ -9,6 +9,10 @@ class HrExpense(models.Model):
 
     @api.multi
     def action_submit_expenses(self):
+        context = self._context or {}
+        if context.get('select_multi_records') and\
```

Comment:
This is smart, but I'm not sure we need it.
It can't think of  a bad case if we don't use this context key.
Thoughts?